comments-bundle/src/DependencyInjection/ContaoCommentsExtension.php

```diff
@@ -22,6 +22,7 @@ class ContaoCommentsExtension extends Extension
     public function load(array $configs, ContainerBuilder $container): void
     {
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../../config'));
+        $loader->load('listener.yaml');
```

Comment:
not sure we really need a new `listener.yaml` file just for one service? @contao/developers wdyt?

Reply:
@aschempp anything regarding this? 🤔

Reply:
I think I agree. The smaller bundles should just have one `services.yaml` containing all the service definitions, no need to organize them (as there won't be that many).